consul/datadog_checks/consul/consul.py

```diff
@@ -72,6 +72,12 @@ class ConsulCheck(AgentCheck):
 
     STATUS_SEVERITY = {AgentCheck.UNKNOWN: 0, AgentCheck.OK: 1, AgentCheck.WARNING: 2, AgentCheck.CRITICAL: 3}
 
+    HTTP_CONFIG_REMAPPER = {
+        'client_cert_file': {'name': 'tls_cert', 'default': None, 'invert': False},
+        'private_key_file': {'name': 'tls_private_key', 'default': None, 'invert': False},
+        'ca_bundle_file': {'name': 'tls_ca_cert', 'default': True, 'invert': False},
```

Comment:
```suggestion
        'ca_bundle_file': {'name': 'tls_ca_cert'},
```

Hum, any reason for using `'default': True` ? seems not a boolean.

Reply:
The config had possible values of `true`, `false` or '<TRUSTED_CA_BUNDLE_FILE_PATH>'. It seems that the wrapper checks if the value is a string and sets `http.options['verify']` to it.